Firmware/Marlin_main.cpp

```diff
@@ -806,7 +806,8 @@ int uart_putchar(char c, FILE *)
 void lcd_splash()
 {
 	lcd_clear(); // clears display and homes screen
-	lcd_printf_P(PSTR("\n Original Prusa i3\n   Prusa Research\n%20.20S"), PSTR(FW_VERSION));
+	lcd_puts_P(PSTR("\n Original Prusa i3\n   Prusa Research\n"));
+	lcd_print_pad_P(PSTR(FW_VERSION), LCD_WIDTH);
```

Comment:
Doesn't this move the text to be left aligned? The old one is right aligned

Reply:
Good catch. The savings here were only minimal. I'll revert this change.